app/models/story.model.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, Sequelize) => {
+    const Story = sequelize.define("story", {
+        userId: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
```

Comment:
I think you need to add an auto incrementing storyId field here: 
```suggestion
  storyId: {
    type: DataTypes.INTEGER,
    primaryKey: true,
    autoIncrement: true
  },
  
  userId: {
     type: Sequelize.STRING,
     allowNull: false,
  },
```